editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -631,8 +631,8 @@ void TileSetEditor::_on_edit_mode_changed(int p_edit_mode) {
 			spin_z_index->hide();
 		} break;
 		case EDITMODE_COLLISION:
-		case EDITMODE_NAVIGATION:
-		case EDITMODE_OCCLUSION: {
+		case EDITMODE_OCCLUSION:
+		case EDITMODE_NAVIGATION: {
```

Comment:
Just out of curiousity, what's this change about? Both lead to the same block of code, so I don't know what switching them around will do.

Reply:
A small consistency change. In the editor, the buttons are organized in "Collision, Occlusion, Navigation".